### DISCUSSION TITLE
MacBook Pro 14 inch - problem with driving high impedance headphones

### DISCUSSION THREAD

**Discussion by mrxiatix** [ExistentialAudio/BlackHole#565] [Q&A]
Hi,
My problem is a quite strange one... I have a pair of 80 ohm dt770pros which I use basically all day with my MacBook Pro. In order to put a systemwide equalizer on the Mac, ,I use BlackHole paired with Hosting AU, where I use the AUNbandEQ. Here is where my problems begin. If as audio output I use 'external headphones' the sound of the headphones is perfectly fine, since the Mac recognises the impedance of  and delivers enough power to drive them, however when I pass the audio through BlackHole, the headphones don't get enough power and they sound empty and not loud enough. How can this be solved?

**Comment by devinroth** [comment]
Is the volume all the way up on BlackHole?

Devin
